be/src/runtime/load_stream.cpp

```diff
@@ -258,19 +259,21 @@ Status LoadStream::init(const POpenStreamSinkRequest* request) {
 Status LoadStream::close(int64_t src_id, const std::vector<PTabletID>& tablets_to_commit,
                          std::vector<int64_t>* success_tablet_ids,
                          std::vector<int64_t>* failed_tablet_ids) {
-    std::lock_guard lock_guard(_lock);
+    std::lock_guard<bthread::Mutex> lock_guard(_lock);
     SCOPED_TIMER(_close_wait_timer);
 
     // we do nothing until recv CLOSE_LOAD from all stream to ensure all data are handled before ack
-    _open_streams[src_id]--;
+    _close_load_cnt++;
```

Comment:
member of LoadStream,std::set LoadStream::_tablets_to_commit;
_tablets_to_commit.add(tablets_to_commit);